src/sparseml/version.py

```diff
@@ -45,7 +49,7 @@ def _generate_version():
     "version_build",
     "version_major_minor",
 ]
-__version__ = _generate_version()
+__version__ = _generate_version(is_release, is_dev, version_base, dev_number)
```

Comment:
Do we need to pass these variables? I think they have been defined in the first part of this file?

Reply:
Yeah we need those parameters, if not they break